ssl/statem/statem_clnt.c

```diff
@@ -3079,44 +3067,48 @@ static int tls_construct_cke_dhe(SSL *s, WPACKET *pkt)
         goto err;
     }
 
-    dh_clnt = EVP_PKEY_get0_DH(ckey);
-
-    if (dh_clnt == NULL) {
-        SSLfatal(s, SSL_AD_INTERNAL_ERROR, SSL_F_TLS_CONSTRUCT_CKE_DHE,
-                 ERR_R_INTERNAL_ERROR);
-        goto err;
-    }
-
     if (ssl_derive(s, ckey, skey, 0) == 0) {
         /* SSLfatal() already called */
         goto err;
     }
 
     /* send off the data */
-    prime_len = BN_num_bytes(DH_get0_p(dh_clnt));
+
+    /* Generate encoding of server key */
+    encoded_pub_len = EVP_PKEY_get1_tls_encodedpoint(ckey, &encoded_pub);
```

Comment:
Travis complains about `EVP_PKEY_get1_tls_encodedpoint`...  missing an include file or has it been renamed by now?

https://travis-ci.com/github/openssl/openssl/jobs/405072122#L3847-L3850

Reply:
This was the result of a merge error when I rebased. Fixed now.